services/usb-device-xous/Cargo.toml

```diff
@@ -19,7 +19,7 @@ num-traits = {version = "0.2.14", default-features = false}
 rkyv = {version = "0.4.3", default-features = false, features = ["const_generics"]}
 susres = {package = "xous-api-susres", version = "0.9.36"}
 modals = {path = "../modals"}
-keyboard = {path = "../keyboard", feature="inject-api"}
+keyboard = { path = "../keyboard", features = ["inject-api"] }
```

Comment:
ah I had just fixed this in a separate patch, before I saw this PR. I'll just drop this one for now...